src/help/faq/README.md

```diff
@@ -19,9 +19,11 @@ Currently an early version of the app is available on the App Store! [Click here
 If you are a patreon, you will recieve an email to your patreon address regarding information about accessing Paperback through TestFlight. The TestFlight version of the application contains many additional beta features which can be used before it becomes available on the App Store.
 
 **NOTE** 
```

Comment:
As there is more than one note, should a plural be used?
```suggestion
**NOTES** 
```

Reply:
"Note" is is being used as a verb in this context for which there is no plural.

Reply:
It's interesting to know. Thanks for the review @GameFuzzy!